extensions/skyportal/static/js/components/Alerts.jsx

```diff
@@ -68,37 +97,324 @@ const useStyles = makeStyles((theme) => ({
+      name: "jd",
+      label: "JD",
+      options: {
+        filter: false,
+        sort: true,
+        sortDescFirst: true,
+        customBodyRender: (value, tableMeta, updateValue) => value.toFixed(5),
+      },
+    },
+    {
+      name: "ra",
+      label: "R.A.",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => ra_to_hours(value, ":"),
+      },
+    },
+    {
+      name: "dec",
+      label: "Decl.",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => dec_to_dms(value, ":"),
+      },
+    },
+    {
+      name: "fid",
+      label: "fid",
+      options: {
+        filter: true,
+        sort: true,
+      },
+    },
+    {
+      name: "magpsf",
+      label: "magpsf",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value.toFixed(3),
+      },
+    },
+    {
+      name: "sigmapsf",
+      label: "sigmapsf",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value.toFixed(3),
+      },
+    },
+    {
+      name: "programid",
+      label: "programid",
+      options: {
+        filter: true,
+        sort: true,
+      },
+    },
+    {
+      name: "isdiffpos",
+      label: "isdiffpos",
+      options: {
+        filter: true,
+        sort: true,
+      },
+    },
+    {
+      name: "drb",
+      label: "drb",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value ? value.toFixed(5) : value,
+      },
+    },
+    {
+      name: "acai_h",
+      label: "acai_h",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value ? value.toFixed(5) : value,
+      },
+    },
+    {
+      name: "acai_n",
+      label: "acai_n",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value ? value.toFixed(5) : value,
+      },
+    },
+    {
+      name: "acai_o",
+      label: "acai_o",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value ? value.toFixed(5) : value,
+      },
+    },
+    {
+      name: "acai_v",
+      label: "acai_v",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value ? value.toFixed(5) : value,
+      },
+    },
+    {
+      name: "acai_b",
+      label: "acai_b",
+      options: {
+        filter: false,
+        sort: true,
+        customBodyRender: (value, tableMeta, updateValue) => value ? value.toFixed(5) : value,
+      },
+    },
+  ];
+
+  const { register: registerForm, handleSubmit: handleSubmitForm, control: controlForm } = useForm();
```

Comment:
Thoughts on defining more custom validation logic on this form to check for "not all position params filled out" errors? It looks like right now, if one of those are left out, the duck will just make an `/api/alerts?objectId=${object_id}` call instead of at least passing along the ones that were filled out, meaning we won't see the more informative error messages from K right? So I think we either do that check in the form validation, or tweak the duck to something like `ra.length || dec.length || radius.length` instead of `ra.length && dec.length && radius.length`

Reply:
Added a check to the `submitSearch` function. The duck shouldn't be checking such stuff and form validation would be too custom, I think.